src/Native/System.Security.Cryptography.Native/pal_pkcs7.cpp

```diff
@@ -12,3 +19,36 @@ extern "C" PKCS7* DecodePkcs7(const unsigned char* buf, int32_t len)
 
     return d2i_PKCS7(nullptr, &buf, len);
 }
+
+extern "C" PKCS7* D2IPkcs7Bio(BIO* bp)
+{
+    return d2i_PKCS7_bio(bp, nullptr);
+}
+
+extern "C" void Pkcs7Destroy(PKCS7* p7)
+{
+    if (p7 != nullptr)
+    {
+        PKCS7_free(p7);
+    }
+}
+
+extern "C" int32_t GetPkcs7Certificates(PKCS7* p7, STACK_OF(X509) * *certs)
```

Comment:
I assume the macro is throwing off clang-format. Perhaps switch to:` typedef STACK_OF(X509) X509Stack` and then use `X509Stack**` here?